sale_order_import/wizard/import_sale.py

```diff
@@ -2,202 +2,480 @@
 class ImportSale(models.TransientModel):
     _name = 'import.sale'
 
     @api.model
-    def _get_picking_policy(self):
+    def _default_picking_policy(self):
+        """Get picking policy"""
         default_rec = self.env['sale.import.default'].search(
             [('company_id', '=', self.env.user.company_id.id)], limit=1)
         if default_rec:
             return default_rec.picking_policy
 
     @api.model
-    def _get_customer_invoice_journal_id(self):
+    def _default_customer_invoice_journal(self):
+        """Get customer invoice journal"""
         default_rec = self.env['sale.import.default'].search(
             [('company_id', '=', self.env.user.company_id.id)], limit=1)
         if default_rec:
             return default_rec.customer_invoice_journal_id
 
     @api.model
-    def _get_customer_payment_journal_id(self):
+    def _default_customer_payment_journal(self):
+        """Get customer payment journal"""
         default_rec = self.env['sale.import.default'].search(
             [('company_id', '=', self.env.user.company_id.id)], limit=1)
         if default_rec:
             return default_rec.customer_payment_journal_id
 
     input_file = fields.Binary(
-        'Sale Order File (.csv Format)',
+        string='Sale Order File (.csv Format)',
         required=True,
     )
     datas_fname = fields.Char(
-        'File Path'
+        string='File Path'
     )
     picking_policy = fields.Selection([
         ('direct', 'Deliver each product when available'),
         ('one', 'Deliver all products at once')],
-        required=True,
         string='Shipping Policy',
-        default=_get_picking_policy,
+        required=True,
+        default=_default_picking_policy,
     )
     customer_invoice_journal_id = fields.Many2one(
         'account.journal',
-        required=True,
         string='Customer Invoice Journal',
-        default=_get_customer_invoice_journal_id,
+        required=True,
+        default=_default_customer_invoice_journal,
     )
     customer_payment_journal_id = fields.Many2one(
         'account.journal',
-        required=True,
         string='Customer Payment Journal',
-        default=_get_customer_payment_journal_id,
+        required=True,
+        default=_default_customer_payment_journal,
     )
 
     @api.model
-    def _get_partner_dict(self, partner_value, partner_dict, error_line_vals):
+    def _get_order_value_dict(self, row, error_line_vals, partner_id,
+                              product_id, pricelist_id, warehouse_id,
+                              team_id, carrier_id, partner_dict, product_dict,
+                              pricelist_dict, picking_dict, warehouse_dict,
+                              team_dict, carrier_dict):
+        """Get order value dict"""
+        partner_value = row[partner_id].strip()
+
+        if not partner_value:
+            error_line_vals['error_name'] = error_line_vals['error_name']\
+                + _('Partner is empty!') + '\n'
```

Comment:
For the all error messages, please apply the following format:
'Column "XXX" cannot be empty.' / 'Column "XXX" cannot be less than 0.'